testlib/resources.go

```diff
@@ -34,11 +34,38 @@ type testResourceDetails struct {
 	action  int8
 }
 
+// commonBaseSearchPaths is a custom version of what fileutils exposes. At some point, consolidate.
```

Comment:
Right now, this is a vanilla copy -- ideally, we'd just recurse to the root for the `utils.FindFile` directly.